R/release.R

```diff
@@ -119,7 +119,15 @@ release_checklist <- function(version, on_cran) {
   )
 }
 
-release_extra <- function(env = parent.env(globalenv())) {
+release_extra <- function(package = NULL) {
+  package <- package %||% project_name()
+  s <- search()
+  m <- match(paste0("package:", package), s)
+  if (is.na(m)) {
+    return(character())
+  }
```

Comment:
I'm not even sure how much we care about the package being attached? But this is the first time I've ever touched `release_extra()`, so maybe it is important?